<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabButtonForm</class>
 <widget class="QWidget" name="TabButtonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>21</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="changedButton">
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="addtoolButton">
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="removetoolButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
